lang/ru.json

```
@@ -0,0 +1,498 @@
+          "71_80": "\\*сдвигаю вас друг к другу*",
+          "61_70": "Похоже на что-то серьезное.",
+          "51_60": "Целуй уже!!!",
+          "41_50": "Уже пара?",
+          "31_40": "У вас есть шансы, чтобы стать парой.",
+          "21_30": "Френдзона+ (Теперь с *плюсиками*)",
+          "11_20": "Френдзона",
+          "1_10": "Похоже даже не друзья...",
+          "0": "Если бы любовь была Солцнем, то это кубик льда."
+        },
+        "special_user": "{user} Авввв дорогуша. Мы и так знаем, что вы всегда идеальная пара на все 100.\nТут даже тест не нужен.",
+        "target_bot": "Боты существуют не для шипперинга, чудак!"
+      },
+      "slap": {
+        "loading": "{LOADING} Шлепаем по картинкам...",
+        "mention_self": "Ненене! Я не позволю шлепать самих себя, {user}.",
+        "mention_purr": "Неет... З-за что, {user}? T^T",
+        "message": "{user} шлепает {targets}",
+        "no_mention": "Пожалуйста упомяните хотя бы одного пользователя, чтобы шлёпнуть!"
+      },
+      "tickle": {
+        "loading": "{LOADING} Щекотим гифку...",
+        "mention_self": "Окей... Если ты этого так хочешь... \\*щекотит {user}*",
+        "mention_purr": "Н-не честно! Я н-не \\*смеется от щекотки*",
+        "message": "{user} щекотит {targets}",
+        "no_mention": "Упомяните, как минимум одного участника для щекотулек!"
+      }
+    },
+    "guild": {
+      "language": {
+        "few_args": "Не достаточно аргументов!\nИспользуйте: `{prefix}language set <язык>`",
+        "invalid_args": "Не правильные аргументы!\nИспользуйте: `{prefix}language <set <язык>|reset>`",
+        "invalid_lang": "Указанный язык не правильный!\nИспользуйте `{prefix}language` чтобы увидеть список доступных языков.",
+        "embed": {
+          "available_lang_title": "Доступные языки",
+          "curr_lang_title": "Установленный язык",
+          "curr_lang_value": "Русский (ru)",
+          "description": "Вы можте изменить язык на один из поддерживающихся, указанных ниже, используя `{prefix}language set <язык>`.",
+          "translators_title": "Переводчики",
+          "translators_value": "Спасибо всем, кто переводил бота!\nПолный список переводчиков и их переводов указан [здесь]({link})."
+        },
+        "language_set": "Язык установлен на `Русский (ru)`!",
+        "no_language": "Напишите язык, который хотите установить!"
+      },
+      "prefix": {
+        "few_args": "Недостаточно аргументов!\nИспользуйте: `{prefix}prefix <set <новый префикс>|reset>`",
+        "invalid_args": "Не правильные аргументы!\nИспользуйте: `{prefix}prefix <set <новый префикс>|reset>`",
+        "no_prefix": "Вам необходимо написать префикс!",
+        "reset": "Установлен стандартный префикс `.`",
+        "set": "Установлен префикс `{prefix}`"
+      },
+      "welcome": {
+        "current_msg": Текущее сообщение:\n```\n{message}\n```\n\nВы можете использовать метки.\n**Метки**:\n`{mention}`\n`{name}`\n`{guild}`\n`{count}`",
```

You forgot a " here
```suggestion
        "current_msg": "Текущее сообщение:\n```\n{message}\n```\n\nВы можете использовать метки.\n**Метки**:\n`{mention}`\n`{name}`\n`{guild}`\n`{count}`",
```

---

Edited!